packages/elasticsearch-plugin/e2e/e2e-helpers.ts

```diff
@@ -63,8 +63,8 @@ export async function testMatchSearchTerm(client: SimpleGraphQLClient) {
         },
     );
     expect(result.search.items.map(i => i.productName)).toEqual([
-        'Instant Camera',
         'Camera Lens',
+        'Instant Camera',
```

Comment:
`Camera Lens` is now the first result because name is more important. In most cases this is desired, but this test case is debatable... WDYT?